packages/fluentui/docs/src/components/ExampleSnippet/ExampleSnippet.tsx

```diff
@@ -1,17 +1,18 @@
-import { CodeSnippet, renderElementToJSX } from '@fluentui/docs-components';
 import * as React from 'react';
+import { CodeSnippet, renderElementToJSX } from '@fluentui/docs-components';
+import { ProviderConsumer } from '@fluentui/react-northstar';
 
 export type ExampleSnippetProps = {
   children?: React.ReactElement;
   render?: () => React.ReactElement;
   value?: string;
 };
 
-const rootStyle = {
-  background: 'white',
+const rootStyle = siteVariables => ({
+  background: siteVariables.bodyBackground,
```

Comment:
I can be a little bias here but `siteVariables.bodyBackground` seems for me it can be anything but in the case it's only colors right? should it be inside `colors`?

Reply:
but should be good when I add also because they go hand-in-hand
`color: siteVariables.bodyColor`